frappe_whatsapp/frappe_whatsapp/doctype/whatsapp_settings/whatsapp_settings.json

```diff
@@ -44,12 +45,17 @@
    "fieldname": "webhook_verify_token",
    "fieldtype": "Data",
    "label": "Webhook Verify Token"
+  },
+  {
+   "fieldname": "bench_location",
```

Comment:
why do we need to add bench location as input? Instead we can directly get it via code right?

Reply:
yes, we can get it via code. I have added a commit to update it.

Please review it.